protocol/app/upgrades/v6.0.0/upgrade.go

```diff
@@ -42,10 +53,62 @@ func removeStatefulFOKOrders(ctx sdk.Context, k clobtypes.ClobKeeper) {
 	}
 }
 
+func setMarketMapParams(ctx sdk.Context, mmk marketmapkeeper.Keeper) {
+	err := mmk.SetParams(ctx, marketmaptypes.Params{
+		// init so that gov is the admin and a market authority
+		MarketAuthorities: []string{GovAuthority /* TODO: add skip market map authority*/},
```

Comment:
nit: Remove the `TODO`, this should be agnostic of any deployment and be set via governance for a deployment.